src/aurora/erfwriter.cpp

```diff
@@ -0,0 +1,108 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Writing BioWare's ERFs (encapsulated resource file).
+ */
+
+#include <ctime>
+#include "erfwriter.h"
+
+namespace Aurora {
+
+ERFWriter::ERFWriter(uint32 id, uint32 fileCount, Common::SeekableWriteStream &stream, LocString description) :
+		_stream(stream), _id(id), _currentFileCount(0), _fileCount(fileCount), _description(description) {
+	stream.writeUint32BE(_id);
+	stream.writeUint32BE(MKTAG('V', '1', '.', '0'));
+
+	// Write Header.
+	stream.writeUint32LE(_description.getNumStrings()); // Language count.
+	stream.writeUint32LE(_description.getWrittenSize()); // Localized string size.
+
+	stream.writeUint32LE(_fileCount); // Entry Count.
+
+	uint32 locStringTableOffset = 160; // 160 is the size of the erf header.
```

Comment:
That's even a static const right there. `static const kLocStringTableOffset = 160;`